charts/operator-wandb/charts/weave-trace/templates/_helpers.tpl

```diff
@@ -4,19 +4,19 @@
 Expand the name of the chart.
 */}}
 {{- define "weaveTrace.name" -}}
-{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- default "weave-trace" .Values.nameOverride | trunc 63 | trimSuffix "-" }}
 {{- end }}
 
 {{/*
 Create a default fully qualified name for weave-trace. (Should be something like wandb-weave-trace)
 We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
 If release name contains chart name it will be used as a full name.
 */}}
-{{- define "weaveTrace.fullname" -}}
+{{- define "weave-trace.weaveTrace.fullname" -}}
```

Comment:
we shouldn't be including weave-traces infront of everything, thats inherited by helm